ethereum/slogging.py

```diff
@@ -138,11 +85,228 @@ def pop_records(self):
 DEFAULT_LOGLEVEL = logging.INFO
 JSON_FORMAT = '%(message)s'
 PRINT_FORMAT = '%(levelname)s:%(name)s\t%(message)s'
+#FORMAT = "%(levelname)s:%(module)s %(asctime)-15s %(message)s"
```

Comment:
Please remove commented out code. If it is needed later it can be retrieved from the git history.